internal/regionsrv/server.go

```diff
@@ -36,28 +36,28 @@ type SuseBuildConfig struct {
 	Ca           string `json:"ca"`
 }
 
-// suseBuildAddress returns a string containing the full address of the TCP
-// server. You may tweak this by providing `SUSE_BUILD_IP` and/or
-// `SUSE_BUILD_PORT`, otherwise `0.0.0.0` and `7956` are taken as defaults for
+// containerBuildSrvAddress returns a string containing the full address of the TCP
+// server. You may tweak this by providing `CONTAINER_BUILD_IP` and/or
+// `CONTAINER_BUILD_PORT`, otherwise `0.0.0.0` and `7956` are taken as defaults for
 // the IP and the port respectively.
-func suseBuildAddress() string {
-	ip := os.Getenv("SUSE_BUILD_IP")
+func containerBuildSrvAddress() string {
+	ip := os.Getenv("CONTAINER_BUILD_IP")
```

Comment:
I can understand the general renaming of it all, by I'm not 100% sold on the idea of changing the environment variables from `SUSE_` to `CONTAINER_`. I like the former better because it specifies that it's something SUSE-specific. Whereas leaving it as `CONTAINER_` might confuse customers thinking that this environment variable is coming from Docker or somewhere else.

Reply:
How strong is your opinion on this? Can we agree to disagree here?  :)

I'd really like to have it consistent with the naming in the service. Also I am not a huge fan of spreading the company name into variable names, might be from the experience of previous acquisitions. I admit this point is a bit mood for a tool call `container-suseconnect` ;-)

Reply:
I have no strong opinions on this. Go as you prefer, it was just a suggestion :)

Reply:
Let's stick with a value that goes after the projects' name. Hence let's stick with the current code